<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>A. Mhamdi</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dashboard USDGE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>441</width>
      <height>261</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_2">
     <widget class="QFrame" name="depot">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>411</width>
        <height>60</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCommandLinkButton" name="startDepot">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>10</y>
         <width>85</width>
         <height>40</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Démarrer le serveur de dépôt des rapports.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(26, 95, 180);</string>
       </property>
       <property name="text">
        <string>Démarrer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stopDepot">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>85</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(220, 50, 47);</string>
       </property>
       <property name="text">
        <string>Arrêter</string>
       </property>
      </widget>
      <widget class="QLabel" name="depot_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>141</width>
         <height>50</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Dépôt des rapports</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="portDepot">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>15</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Delicious Heavy'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;PORT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="jury">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>411</width>
        <height>60</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="stopJury">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>85</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(220, 50, 47);</string>
       </property>
       <property name="text">
        <string>Arrêter</string>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="startJury">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>10</y>
         <width>85</width>
         <height>40</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Affecter les membres de jury.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(26, 95, 180);</string>
       </property>
       <property name="text">
        <string>Démarrer</string>
       </property>
      </widget>
      <widget class="QLabel" name="jury_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>141</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>Affectation du jury</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="portJury">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>15</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Delicious Heavy'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;PORT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="soutenance">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>411</width>
        <height>60</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="stopSoutenance">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>85</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(220, 50, 47);</string>
       </property>
       <property name="text">
        <string>Arrêter</string>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="startSoutenance">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>10</y>
         <width>85</width>
         <height>40</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Démarrer les soutenances.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(26, 95, 180);</string>
       </property>
       <property name="text">
        <string>Démarrer</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="portSoutenance">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>15</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Delicious Heavy'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;PORT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="soutenance_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>141</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>Portail d'évaluation</string>
       </property>
      </widget>
     </widget>
     <zorder>soutenance</zorder>
     <zorder>jury</zorder>
     <zorder>depot</zorder>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QFrame" name="email">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>20</y>
        <width>421</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Zipper et envoyer e-mails</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="init">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>10</y>
         <width>41</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(249, 240, 107);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>I</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="perf">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>10</y>
         <width>41</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(38, 162, 105);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>P</string>
       </property>
      </widget>
      <widget class="QPushButton" name="envoyer">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>10</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>421</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QCalendarWidget" name="calendarWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>421</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="email_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>420</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Générer les pvs</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="init_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>10</y>
         <width>41</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(249, 240, 107);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>I</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="perf_2">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>10</y>
         <width>41</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(38, 162, 105);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>P</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>10</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:10pt; font-style:italic; color:#1a5fb4;&quot;&gt;ISET de Bizerte&lt;br/&gt;Département de génie électrique&lt;br/&gt;Unité des stages&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>350</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/ISETBZ.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Montserrat','sans-serif'; font-size:36px; color:#ffffff; text-transform:uppercase;&quot;&gt;Gestion des stages &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
